docs/release_notes.md

```diff
@@ -37,6 +37,10 @@
 - Fixed a bug in the Volume Constraint that was causing flickering if the simulated geometry was far from the center of the world. *AdonisFX-1489*
 - Fixed the internal bounding box construction at start time to use the current point positions instead of the point positions at rest. *AdonisFX-1511*
 
+### Deprecated
+
+- The AdnWeightsDisplayNode has been deprecated. This deprecation is fully backward-compatible, ensuring that scenes created in earlier versions will work as expected in version 1.4.0. To safely remove deprecated nodes from your scenes, use the utility available in AdonisFX Menu > Utils > *Upgrade Deprecated Nodes*.
```

Comment:
As this page contains the release notes, it is not a page to explain "how to" use a too. I would rewrite this slightly:
```suggestion
- The AdnWeightsDisplayNode is deprecated. This deprecation is fully backward-compatible, ensuring that scenes created in earlier versions will work as expected in version 1.4.0. To safely remove deprecated nodes from your scenes, a utility is added in AdonisFX Menu > Utils > *Upgrade Deprecated Nodes*.
```